http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitorSpec.scala

```diff
@@ -101,6 +101,7 @@ class BaseCloudServiceRuntimeMonitorSpec extends AnyFlatSpec with Matchers with
       disk <- makePersistentDisk().save()
       start <- IO.realTimeInstant
       tid <- traceId.ask[TraceId]
+      implicit0(ec: ExecutionContext) = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
What does his do? Still wrapping my head around how to use implicits properly 😭

Reply:
...honestly I'm not entirely sure myself, just that I needed the EC implicit to do the disk query